consensus/bor/bor.go

```diff
@@ -1191,11 +1194,15 @@ func (c *Bor) CommitStates(
 		}
 		stateSyncs = append(stateSyncs, &stateData)
 
-		if err := c.GenesisContractsClient.CommitState(eventRecord, state, header, chain); err != nil {
+		gasUsed, err := c.GenesisContractsClient.CommitState(eventRecord, state, header, chain)
+		if err != nil {
 			return nil, err
 		}
+		totalGas += int(gasUsed)
+
 		lastStateID++
 	}
+	log.Info("StateSyncData", "Gas", totalGas, "Block-number", number, "LastStateID", lastStateID, "TotalRecords", len(eventRecords))
```

Comment:
I think I am more interested in individual gas usage instead of total.

Reply:
Adding them !

Reply:
I did not want to change the struct of Event and make many changes in the consensus module, so added the functionality without disturbing the structs.